openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -576,9 +595,12 @@
     "select-a-chart": "Select a Chart",
     "select-a-metric-type": "Select a Metric Type",
     "select-a-policy": "Select a Policy",
+    "select-add-test-suite": "Select/Add Test Suite",
     "select-column-plural-to-exclude": "Select Columns to Exclude",
     "select-column-plural-to-include": "Select Columns to Include",
     "select-field": "Select {{field}}",
+    "select-teams": "Select teams",
+    "select-type": "Select type",
```

Comment:
We have `select-field` present, can we use that?

Reply:
I will cover that up when I raise PR to optimise en-US.json file itself